src/spikeinterface/preprocessing/detect_bad_channels.py

```diff
@@ -17,8 +17,8 @@ def detect_bad_channels(
     n_neighbors=11,
     nyquist_threshold=0.8,
     direction="y",
-    chunk_duration_s=0.3,
-    num_random_chunks=10,
+    chunk_duration_s=0.5,
+    num_random_chunks=100,
```

Comment:
I would just increase the number of random chunks.

Reply:
The noisy channel are done on PSD. 300ms looks like quite short. I wanted to increase a bit.
where this 300ms comme from ?

Reply:
Hey both, this was the default from the IBL white paper (code [here](https://github.com/int-brain-lab/ibl-neuropixel/blob/8a8cd834877167eee573ce0ae6b3fdfcbcd05bbc/src/neurodsp/voltage.py#L625)). In the context of the IBL pipeline PSD (I am not sure for the other methods in `detect_bad_channels`), I do not think changing this would make too much difference. At 30 kHz sampling rate the FFT frequency bin-width of 300 ms trace would be 30k / 9k = 3.33 Hz vs. 2 Hz (using 500 ms). The main use of the PSD in the IBL function is to calculate the average power above 80% Nyquist. For example, the true 80% of Nyquist at 30 kHz is the range 12-15000 Hz. This range would be something like 11997.99-14998.32 in practice with 3.33 Hz bins. So I don't think in this context at least it should matter much but I might have made an oversight / mistake.